libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.html

```diff
@@ -38,11 +38,10 @@
           }}
         </option>
       </select>
-    </sky-input-box>
-    <sky-form-errors [labelText]="labelText">
       <sky-form-error
         *ngIf="
-          hostControl?.errors?.['skyDateRange']?.errors.endDateBeforeStartDate
+          hostControl?.errors?.['skyDateRange']?.errors
+            .endDateBeforeStartDate && labelText
```

Comment:
Is `&& labelText` needed since it's being wrapped in the input box now?

Reply:
yup!